boards/msb-430/include/board.h

```diff
@@ -43,8 +43,8 @@ extern "C" {
 /**
  * @brief   Define the CPU model for the <msp430.h>
  */
-#ifndef __MSP430F1612__
-#define __MSP430F1612__
+#ifndef MSP430F1612
```

Comment:
Ah, I see... it seems the all the related boards here are MSP430 series.. So, maybe I shouldn't revise them.. I think I should close this PR, right? :-)  @authmillenon